fs/tmpfs/fs_tmpfs.c

```diff
@@ -55,6 +55,11 @@
 #  warning CONFIG_FS_TMPFS_FILE_FREEGUARD needs to be > ALLOCGUARD
 #endif
 
+#define tmpfs_lock(fs) nxrmutex_lock(&(fs)->tfs_lock)
```

Comment:
Optional, but now seems that all wrappers can be just replaced with `nxrmutex_lock`/`nxrmutex_unlock`, so we do not need defines at all.